### DISCUSSION TITLE
Practicas de la parte front end

### DISCUSSION THREAD

**Discussion by AlexChris23** [Launch-X-Latam/MisionFrontEnd#124] [Q&A]
Hola buen dia, estoy un poco perdido aunque he visto las grabaciones se que es primero ir haciendo bocetos pero, hay plantillas de practicas las cuales tengo dudas.

¿Las palntillas se van a contestaron y/o realizar? 
Si es así en alguna clase darán ejemplos de como hacerlas porque estoy un poco perdido en ello

**Comment by iam-leo** [comment]
:large_blue_diamond: Hola compa! Al respecto de tu pregunta, el MC nos dice siempre que esta es una experiencia autodidacta, con lo cual dudo mucho que tengamos una clase específica de como llenar las plantillas o de como usar Figma por ejemplo. 

:large_blue_diamond: Las clases son una pequeña guía, el resto es investigar por cuenta propia y/o apoyarse en la comunidad.

:large_blue_diamond: Debes tener las prácticas realizadas, aunque sea a tu manera, ya que si luego nos piden una entrega o hacer un repositorio con tales archivos, ya tendrás la práctica realizada. 

:large_blue_diamond: Hasta el momento no nos dijeron nada al respecto de si hay que entregar las prácticas o no, en caso de que si haya que entregar las actividades puede ser para una determinada fecha o al finalizar la misión, pero ten en cuenta que si piden todas las prácticas juntas al final y no las llevas al día, se te juntarán los pendientes y ya dependerá del tiempo que dispongas para poder concluirlas.
